base/compiler/compiler.jl

```diff
@@ -15,14 +15,21 @@ const swapproperty! = Core.swapfield!
 const modifyproperty! = Core.modifyfield!
 const replaceproperty! = Core.replacefield!
 
+const _included_files = Array{String, 1}()
+include(x) = include(Compiler, x)
+function include(mod, path)
+    if mod === Compiler
+        ccall(:jl_array_grow_end, Cvoid, (Any, UInt), _included_files, UInt(1))
```

Comment:
Could use a comment about what this is doing and why?

Reply:
this is just a `push!` before `push!` is defined.

Reply:
I know that. But I'm sayin' for others :)

Reply:
Agreed that a comment would be nice. `Base.jl` should have had one for the same code too.